### PR TITLE
[16.0][MIG] purchase_force_invoiced

### PR DESCRIPTION
**Pull request by alexis-via**
2 small improvements:
1. force_invoiced now has tracking=True (important !)
2. force_invoiced in now invisible on RFQs

### CODE REVIEW

**Comment by alexis-via** [comment]
Tests are green, cool !

File: purchase_force_invoiced/tests/test_purchase_force_invoiced.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            {
+                "name": name,
+                "categ_id": product_ctg.id,
+                "type": "service",
+                "purchase_method": "receive",
+            }
+        )
+        return product
+
+    def create_invoice_line(self, line, invoice):
```
Comment (StefanRijnhart) [inline purchase_force_invoiced/tests/test_purchase_force_invoiced.py]: Looks like you can remove this method now.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@StefanRijnhart Could you update your review?

**Review by StefanRijnhart — APPROVED** [review]

**Comment by RodrigoBM** [comment]
next time link to migration issue please #1585

**Pull request merged**